server/app.js

```diff
@@ -0,0 +1,14 @@
+const express = require('express')
+const app = express()
+const cors = require('cors');
+const port = 4000
+
+app.use(cors());
+const resultsRoutes = require('./controllers/router')
+app.use('/results', resultsRoutes);
+
+app.get('/', (req, res) => res.send('Server Homepage'))
+
+app.listen(port, ()=>{
+    console.log(`Express departing from http://localhost:${port}`)
+})
```

Comment:
Could split this file into an `app.js` and `index.js` so that the server can be used for testing (if implemented in the future).